interfaces/builtin/core-support.go

```diff
@@ -0,0 +1,131 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
```

Comment:
Files should be named "core_support.go" and "core_support_test.go" (see the other files in the same dir).